papers/ring_vrf/pedersen_vrf.tex

```diff
@@ -118,27 +121,31 @@ \section{Ring VRFs from the Pedersen VRF}
 \item $\rVRF.\rSign\eprint{ : }{}(\sk,\openring,\msg,\aux)\eprint{ \mapsto \rho}{}$
 	returns $\rho = (\compk,\piring,\sigma)$ \eprint{where \\}{\\ where} 
 	\tmpindent $(\openpk,\compk) \leftarrow \PedVRF.\CommitKey$,  \\
-	\tmpindent $\piring \leftarrow \NIZK_{\Lring}.\Prove(\compk,\comring,\openpk,\openring)$, \\
+	\tmpindent $\piring \leftarrow \NIZK_{\Lring}.\Prove((\openpk,\openring),(\compk,\comring))$, \\
```

Comment:
why the grouping?

Reply:
If you don't use relation, it is the only way to show what is the witness in this proof i.e., first component is the statement and the second component is the witness

Reply:
As I said before, the actual "witness" include many additional wires representing intermediate states of computations.  These wires are secret inputs.  An existential quantifier makes them easier to read too.

Reply:
You should separate witness and statement in the Prove algorithm, otherwise zk definition does not make sense.